config/dictionaries/resource.json

```diff
@@ -288,14 +313,34 @@
       {
         "id": "move-device",
         "summary": "Move device to another application",
+        "description": "",
         "method": "PATCH",
         "endpoint": "/v5/device(<ID>)",
         "filters": "",
         "data": "{\n    \"belongs_to__application\": \"<APP ID>\"\n}"
       },
+      {
+        "id": "get-target-release",
+        "summary": "Get the release device is pinned to",
+        "description": "Note: should_be_running__release will output unset or null if the device isn't pinned to any release. For an unpinned device, the device.is_on__commit field may be used to obtain the device's current state, while the application.commit field represents the device's target state.",
+        "method": "GET",
+        "endpoint": "/v5/device(<ID>)",
+        "filters": "?\\$select=should_be_running__release",
+        "data": ""
+      },
+      {
+        "id": "get-current-release",
+        "summary": "Get current release running on device",
+        "description": "The is_on__commit field fetches the hash of the current release that the device is running on",
```

Comment:
* NIT: _"fetches"_ sounds a bit odd to me. How about _"contains"_ instead?  
(An application may fetch some data when it is executed, but a field by itself does not fetch anything...) :-)
* Instead of: _"the current release that the device is running on"_  
I'd suggest: _"the release currently deployed to the device"_  
(A device "runs on an ARM CPU", rather than a device "runs on a release"... I think!)

That's focusing on the writing only, not on whether it is true that `is_on__commit` regards current state rather than target state. (Is `is_on__commit` updated by the supervisor?)

Reply:
I second that for "contains" over "fetches". These are declarative *properties* not computational results (even if technically everything is a computational result 😅)

Reply:
Thanks, resolving!